genie/input_to_database.py

```diff
@@ -165,6 +166,109 @@ def _get_status_and_error_list(valid, message, entities):
     return input_status_list, invalid_errors_list
 
 
+def get_ancillary_files(
```

Comment:
@rxu17 I didn't realize this until right now, another option is to pass in the list of lists from the `get_center_files` to use in this function.

There is quite a bit of added complexity here that I'm just not sure the code is ready to handle yet due to not having `sample` and `patient` classes.

Reply:
In the end, decided to comment out this function and instead use the `extract.get_center_files`.